lib/auth/keystore/gcp_kms.go

```diff
@@ -263,6 +269,14 @@ func (g *gcpKMSKeyStore) DeleteUnusedKeys(ctx context.Context, usedKeys [][]byte
 		return trace.Wrap(err)
 	}
 
+	for _, found := range activeKmsKeyVersions {
+		if found == 0 {
+			// Failed to find a currently active key owned by this host.
+			// The cluster is in a bad state, refuse to delete any keys.
+			return trace.NotFound("Failed to find currently active CA key in GCP KMS, aborting attempt to delete unused keys")
```

Comment:
nit: lowercase error, avoid words like "failed"

```suggestion
			return trace.NotFound("cannot find currently active CA key in GCP KMS, aborting attempt to delete unused keys")
```

Also, do you think it's worth returning or logging some sort of key identifier for the "bad" keys?